plugins/permission-node/api-report.md

```diff
@@ -156,7 +158,9 @@ export interface PermissionPolicy {
   // (undocumented)
   handle(
     request: PolicyQuery,
-    user?: BackstageIdentityResponse,
+    user?: BackstageIdentityResponse<
```

Comment:
😬

Reply:
OK so then given the above, the question is what to do about this signature.

In the end we probably will want to take policy decisions based on service identities too. But we're not quite there yet right? So I'm thinking, for now, would it be better to revert this change? And then in all of the places that call `handle`, it's been checked first that it's actually a user identity.